gatsby-config.js

```diff
@@ -274,8 +274,6 @@ module.exports = {
                         type: `image/png`,
```

Comment:
For this file, please compare this with the current gatsby-config file we have and update the changes we need